### PR TITLE
S-95128 In xl-cli install replace support for XRR with RRR helm chart

### PR DESCRIPTION
**Pull request by vpugar-digital**

### CODE REVIEW

File: xl-op/blueprint.yaml [hunk 1]
```diff
@@ -128,7 +128,7 @@ spec:
       overrideDefault: true
       promptIf: !expr "ServerType == 'dai-release-runner' && (ProcessType == 'install' || ProcessType == 'upgrade')"
       prompt: "Enter the image tag (eg: <tagName> from <repositoryName>/<imageName>:<tagName>):"
-      default: 0.1.33
+      default: v23.3.0-409.1453-tls
```
Comment (d-venkat) [inline xl-op/blueprint.yaml:131]: to check if mentioned version is okay.

File: xl-op/blueprint.yaml [hunk 2]
```diff
@@ -542,7 +542,7 @@ spec:
       overrideDefault: true
       prompt: !expr "IsCustomImageRegistry ? 'Enter the operator image to use (eg: <imageRegistryName>/<repositoryName>/<imageName>:<tagName>):' : 'Enter the operator image to use (eg: <repositoryName>/<imageName>:<tagName>):'"
       promptIf: !expr ServerType == 'dai-deploy' && (ProcessType == 'install' || ProcessType == 'upgrade')
-      default: xebialabsunsupported/deploy-operator:23.3.0-819.1055
+      default: xebialabsunsupported/deploy-operator:23.3.0-906.858
```
Comment (d-venkat) [inline xl-op/blueprint.yaml:545]: maybe this is not intended to be changed. and is left as per master branch update.

File: xl-op/blueprint.yaml [hunk 3]
```diff
@@ -551,7 +551,7 @@ spec:
       overrideDefault: true
       prompt: !expr "IsCustomImageRegistry ? 'Enter the operator image to use (eg: <imageRegistryName>/<repositoryName>/<imageName>:<tagName>):' : 'Enter the operator image to use (eg: <repositoryName>/<imageName>:<tagName>):'"
       promptIf: !expr ServerType == 'dai-release' && (ProcessType == 'install' || ProcessType == 'upgrade')
-      default: xebialabsunsupported/release-operator:23.3.0-819.1130
+      default: xebialabsunsupported/release-operator:23.3.0-906.858
```
Comment (d-venkat) [inline xl-op/blueprint.yaml:554]: maybe this is not intended to be changed. and is left as per master branch update.

**Review by d-venkat — APPROVED** [review]

File: xl-op/blueprint.yaml (hunk 2 above)
Comment (vpugar-digital) [reply xl-op/blueprint.yaml]: Updated to the latest master image build

File: xl-op/blueprint.yaml (hunk 3 above)
Comment (vpugar-digital) [reply xl-op/blueprint.yaml]: Updated to the latest master image build

File: xl-op/blueprint.yaml (hunk 1 above)
Comment (vpugar-digital) [reply xl-op/blueprint.yaml:131]: I have put the image on docker.io/xebialabsunsupported/release-remote-runner

**Pull request merged**